plugin/pkg/global/extensionvalidation/admission.go

```diff
@@ -150,34 +151,78 @@ func (e *ExtensionValidator) Validate(ctx context.Context, a admission.Attribute
 	case core.Kind("BackupBucket"):
```

Comment:
Instead of switching over `a.GetKind().GroupKind()` would about using a [type switch](https://tour.golang.org/methods/16)?